README.md

```diff
@@ -34,6 +34,18 @@ const attachment: Attachment = {
 }
 ```
 
+### Create a preview attachment object
```

Comment:
Added this per your Notion doc, @daria-github! May I ask - should we also add the install step to this README? 

### Install the package

```bash
npm i xmtp-content-type-remote-attachment
```

Reply:
I think that's a great idea. Thanks, J-Ha!